runtests.py

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env python
```

Comment:
Do we need this file? I was playing around with the environment. Pytest will discover all of our tests automatically with `py.test`. In fact, we don't even need to have the `PYTHONPATH` set for it to properly discover. There had been issues with this in past projects, but this project's configuration, it seems to be a non-issue.